### PR TITLE
added preSubmitAccount for bijouxfantaisie specific calculation

### PR DESCRIPTION
**Pull request by vinyll**

### CODE REVIEW

**Comment by vinyll** [comment]
Note that settings `action="javascript:calculator.amount() <= 150…"` would send that exact string as a URL. It seems not to be evaluated within a _action_ attribute.

File: bijouxfantaisie/index.html [hunk 1]
```diff
@@ -136,4 +136,10 @@
   }
 </script>
 <script src=/js/main.js></script>
+<script>
+  function preSubmitAmount(event) {
```
Comment (davidbgk) [inline bijouxfantaisie/index.html]: Missing space before parenthesis.

File: bijouxfantaisie/index.html [hunk 2]
```diff
@@ -136,4 +136,10 @@
...
+    event.target.action = calculator.amount() <= 150
```
Comment (davidbgk) [inline bijouxfantaisie/index.html]: It's actually great to have that specificity within the page and not in our main js.

File: js/main.js [hunk 3]
```diff
@@ -117,9 +117,10 @@ function showPage (page) {
 
 function submitAmount (event) {
   event.preventDefault()
+  if (typeof(preSubmitAmount) !== 'undefined') preSubmitAmount(event)
```
Comment (davidbgk) [inline js/main.js:120]: Not fond of the naming but I can't find a better proposition, I can live with it.

**Review by davidbgk — APPROVED** [review]

**Pull request merged**